From e4421d697d889d1a802babfbd1d0839bc92884a8 Mon Sep 17 00:00:00 2001
From: Vasily Khoruzhick <anarsoul@gmail.com>
Date: Thu, 31 Aug 2017 21:53:07 -0700
Subject: [PATCH 247/256] vidconsole: flush dcache on putc

Otherwise characters that are being typed are not displayed until
user presses "enter".

Signed-off-by: Vasily Khoruzhick <anarsoul@gmail.com>
---
 drivers/video/vidconsole-uclass.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/video/vidconsole-uclass.c b/drivers/video/vidconsole-uclass.c
index b5afd72227..44c716c665 100644
--- a/drivers/video/vidconsole-uclass.c
+++ b/drivers/video/vidconsole-uclass.c
@@ -163,6 +163,7 @@ static void vidconsole_putc(struct stdio_dev *sdev, const char ch)
 	struct udevice *dev = sdev->priv;
 
 	vidconsole_put_char(dev, ch);
+	video_sync(dev->parent);
 }
 
 static void vidconsole_puts(struct stdio_dev *sdev, const char *s)
-- 
2.13.6

